storage/container_storage_interface/persistent-storage-csi-ibm-powervs-block.adoc

```diff
@@ -0,0 +1,33 @@
+[id="persistent-storage-csi-ibm-powervs-block"]
+= {ibmpowerProductName} Virtual Server Block CSI Driver Operator
+include::_attributes/common-attributes.adoc[]
+:context: persistent-storage-csi-ibm-powervs-block
+
+toc::[]
+
+:_content-type: CONCEPT
```

Comment:
According to our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata), the `content-type` needs to be on line 1. Also, this needs to be `ASSEMBLY` not `CONCEPT`.

Reply:
removed : 
:_content-type: CONCEPT

@sheriff-rh please check if correct. Thanks.

Reply:
@alishaIBM content-type needs to be on line 1 and it needs to be ASSEMBLY type: 

```
:_content-type: ASSEMBLY
[id="persistent-storage-csi-ibm-powervs-block"]
= {ibmpowerProductName} Virtual Server Block CSI Driver Operator
include::_attributes/common-attributes.adoc[]
:context: persistent-storage-csi-ibm-powervs-block

toc::[]
```

Reply:
@sheriff-rh 
Incorporated above suggestion. Thanks.